docs/game_sdk/Networking.md

```diff
@@ -206,51 +229,89 @@ A quick note here may be helpful for the two functions that should be called con
 
 `Flush()` is specific to the network manager. It actually _writes_ the packets out to the stream. You should call this at the _end_ of your game loop as a way of saying "OK, I'm done with networking stuff, go send all the stuff to people who need it". In Unity, for example, this goes in `LateUpdate()`.
 
+## Connecting to Each Other
+
+This manager is built around the concept of routes between players, and then channels on those routes. Player A opens a route to Player B. This route will often change as our scaled networking system works to find the best networking path between those players. As that route changes, the player will receive `OnRouteUpdate` events. They should then alert other lobby members that their route has changed by updating their lobbymetadata. Other lobby members will see those updates from the `OnLobbyMemberUpdate` event, and can call `UpdateRoute()` accordingly. A user's route could change frequently, so architect your system anticipating frequent route changes and updates.
+
+Once Player A has a route open to Player B, Player A then opens a channel to Player B, and Player B does the same to Player A. Channels are the pipes down which data is actually sent. These two users can now send data back and forth to each other with `SendMessage()` and receive it with `OnMessage`.
+
+In order to properly send and receive data between A and B, both users need to have **the same type of channel** open to each other **on the same channel number**. If A has Reliable Channel 4 open to B, B also needs Reliable Channel 4 open to A. There are many ways to keep track of this; utilizing metadata on the lobby is one great way.
```

Comment:
I would drop the recommendation to use lobby metadata for this -- in general games should be deterministically opening the same channels on the same channel IDs with the same types for every connection, so they shouldn't need to "track" which channels have been opened and what kind they are.